code/datums/traits/negative.dm

```diff
@@ -576,3 +576,38 @@
 	H.remove_language(/datum/language/common)
 	if(!H.can_speak_in_language(/datum/language/draconic) && !H.can_speak_in_language(/datum/language/machine))
 		H.grant_language(/datum/language/japanese)
+
+/datum/quirk/random_accent
```

Comment:
Perhaps you can make this a child of the /datum/quirk/accent? Someone else comment on this

Reply:
I am commenting on this

Reply:
Then do it

Reply:
maybe *sniff* maybe

Reply:
:)